src/ocm/components/clusterWizard/ClusterWizardFooter.tsx

```diff
@@ -33,6 +34,9 @@ const ValidationSection = ({
   const { currentStepId } = useClusterWizardContext();
   const validationsInfo = cluster && selectClusterValidationsInfo(cluster);
   const { t } = useTranslation();
+  const isPlatformIntegrationFeatureEnabled = useFeature(
```

Comment:
This is not the right condition, here you are looking only at the feature flag.
PTAL at https://github.com/openshift-assisted/assisted-ui-lib/blob/02ba67a9bfa86dbd9320e91445c1a7512bac97eb/src/ocm/components/clusterConfiguration/HostInventory.tsx